backend/.gitignore

```
@@ -0,0 +1 @@
+/config/.env
```

since your build task always generates `server`, i would add so engineers don't commit the binary
```suggestion
/config/.env
server
```